sheet-to-json_generic.js

```diff
@@ -1,16 +1,15 @@
 const { task, fetchData, writeData } = require("./lib");
 const c = require("./lib/constants");
-
+const moment = require("moment");
 
 (async function main() {
   console.log("Running task on start...");
-  await task({
-    sheet: c.SHEET,
-    tabs: {
-      raw_data: c.SHEET_RAW_DATA
-    },
-    file: c.FILE_RAW_DATA
-  });
+  
+  var data = await fetchData({sheet:c.SHEET, tabs:{
+    raw_data: c.SHEET_RAW_DATA
+  }});
+  data.last_updated_time = moment().utcOffset(330) + "";
```

Comment:
Since the time is in epoch, `utcOffset` doesn't do anything here no?
This line can just be like below? 
`data.last_updated_time = moment() + "";`

Reply:
@JunaidBabu I assumed that IST time must be fetched, since as per `raw_data-to-state_district_wise_data.js`, the code reads:

```
const todaysdate = moment().utcOffset(330); //set moment instance to IST timezone and fetch currect IST time
```

Reply:
But both `const todaysdate = moment();` and `const todaysdate = moment().utcOffset(330);` will return the same value right? The time in epoch.

Reply:
Okay @JunaidBabu . Will make the requested correction since both return the same value as you've stated.